models/clm/tutorial/README.rst

```diff
@@ -0,0 +1,1567 @@
+  Year 2011, ensemble members 1-5 only: ``f.e21.FHIST_BGC.f09_025.CAM6assim.011.cpl_000{1-5}.ha2x3h.2011.nc`` 
+  
+  2. `Reference Case and Observations <https://www.image.ucar.edu/pub/DART/CESM/clmdart_tutorial/>`__ 
+   
+
+Step 1: Download CLM5
+---------------------
+
+CLM is continually being updated by the model developer and user community
+consisting of both NCAR and university scientists and researchers.
+In contrast, DART is maintained by a relatively small group that supports
+numerous earth system models (20+) including CLM. Therefore the DART team
+focuses on only supporting official released versions of CLM.  This documentation
+and scripting was tested using the CESM tag ``release-cesm2.2.0`` following
+the download instructions `here <https://github.com/ESCOMP/CESM>`__.
+
+Although the DART code may work with more recent versions of CESM (CLM) we recommend
+checking out ``release-cesm2.2.0``.
+
+  ::
+
+    git clone https://github.com/escomp/cesm.git cesm_dart
+    cd cesm_dart
+    git checkout release-cesm2.2.0
+    ./manage_externals/checkout_external
+
+
+Adding CLM5 SourceMods
+----------------------
+
+Some minor modifications have to be made to the CLM5 source code in order
+to be run with DART. Most importantly, these include skipping several
+balance checks in CLM5 for the time step immediately after the assimilation
+update step.  These sourcecode modifications are brought in 
+through the SourceMod mechanism in CLM where modifications overwrite
+the template sourcecode during the compilation step. The SourceMods
+are located as tar files `here. <http://www.image.ucar.edu/pub/DART/CESM>`__
+For this tutorial use the most recent tar file ``DART_SourceMods_cesm2_2_0_2021_07_02.tar``
+and untar it on your local machine.  
+
+::
+ 
+  tar -xvf DART_SourceMods_cesm2_2_0_2021_07_02.tar
+
+For more information on the 
+SourceMods see the main :doc:`CLM-DART documentation. <../readme>`
+
+
+
+Step 2: Download DART
+--------------------- 
+
+The tutorial material is available within the most recent release of the
+DART repository on the `main branch <https://github.com/NCAR/DART>`__.
+
+::
+ 
+  cd /glade/work/$USER/
+  git clone https://github.com/NCAR/DART.git
+  cd DART
+
+
+
+
+Step 3: Navigating DART Scripts
+-------------------------------
+
+
+Below is a table of the key CLM5-DART setup scripts that include the 
+settings required to perform an  assimilation run. All scripts are 
+located at path ``DART/models/clm/shell_scripts/cesm2_2/`` with the 
+exception of ``input.nml`` which is located at ``DART/models/clm/work/``.
+
+You will be asked to locate, edit, and execute these scripts during the tutorial
+and are they are critical to customizing the assimilation run.  Given their importance, we
+introduce them right away.
+
+For additional description of the CLM5-DART scripts and concepts please
+visit the main :doc:`CLM-DART documentation <../readme>`
```

Comment:
So, all links that start with `:doc` do not take you anywhere. This is done in purpose. Right?

Reply:
Some of the :doc: links needed fixing from the when the PR was first issued, but my commits since the original PR should have fixed this.    I think all the links, including the :doc: links which give a relative path should be working now.   The current github compiled version seems to work OK for this particular link:   https://dart-documentation--330.org.readthedocs.build/en/330/models/clm/tutorial/README.html